molgenis_documentation/user_documentation.md

```diff
@@ -494,7 +494,51 @@ In the following paragraphs we explain on how to use different modules to config
 The home page is the first thing a visitor sees. It should be the thing that informs a user about whatever is on the page he or she has navigated to. It should also sell the content, making it a very important piece
 
 #### <a name="user-management"></a> User management
-User management
+User management is crucial for keeping an overview of people visiting your research database, but it is also important for security reasons. MOLGENIS has an extensive user management system, allowing people to register themselves, or be registered by an administrator. You can find the User manager module under the Admin menu. Going there, you will find something looking like this:
```

Comment:
I'd like a small explanation of the User <-> Group <-> Permission entities that you're managing here.